lib/pat.c

```diff
@@ -92,6 +93,12 @@ typedef struct {
 #define PAT_DELETING  (1 << 1)
 #define PAT_IMMEDIATE (1 << 2)
 
+static inline bool
+pat_key_is_large_total_key_size(grn_pat *pat)
```

Comment:
Can we simplify this?

```suggestion
pat_is_key_large(grn_pat *pat)
```